src/frontends/onnx/tests/__init__.py

```diff
@@ -147,7 +147,7 @@ def xfail_test(reason="Mark the test as expected to fail", strict=True):
 skip_dynamic_model = pytest.mark.skip(reason="CPU plug-in can't load a model with dynamic output shapes via legacy API")
 
 # ONNX 1.14
-xfail_issue_119896 = xfail_test(reason="Unsupported element type: FLOAT8")
+xfail_issue_119896 = xfail_test(reason="Unsupported element type: FLOAT8", strict=False)
```

Comment:
As fp8 conversion is supported, some of the onnx test cases https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/tests/tests_python/test_backend.py#L484-L528 don't need to xfail anymore. For example `test_cast_FLOAT8E4M3FNUZ_to_FLOAT_cpu`. It can get correct result from CPU infer now. However, for other test cases, e.g., `test_cast_FLOAT_to_FLOAT8E4M3FN_cpu`, xfail must be retained because they still fail. For these test cases the expected output data type is f8, but the actual output data type is f32. I've checked CPU plugin delivers output data of f8 correctly,  but it is converted to f32 again by frontend https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/tests/runtime.py#L164. **So I believe we still need modification from frontend side, before we can remove these xfail**. For now I just set strict=False to achieve unified xfail behavior for upconvert and downconvert.